metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -161,11 +185,22 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
         default=False,
         description="Whether to convert the PowerBI assets urns to lowercase",
     )
+
     # convert lineage dataset's urns to lowercase
     convert_lineage_urns_to_lowercase: bool = pydantic.Field(
         default=True,
         description="Whether to convert the urns of ingested lineage dataset to lowercase",
     )
+    # Configuration for stateful ingestion
+    stateful_ingestion: Optional[StatefulStaleMetadataRemovalConfig] = pydantic.Field(
+        default=None, description="PowerBI Stateful Ingestion Config."
+    )
+    # Retrieve PowerBI Metadata using Admin API only
+    admin_apis_only: bool = pydantic.Field(
+        default=False,
+        description="Retrieve metadata using PowerBI Admin API only. If this flag is 'true' then Report's pages "
```

Comment:
"If this is enabled, then Report Pages will not be extracted. Admin API access is required if this setting is enabled"

Reply:
done